inst/pages/multiassay_ordination.qmd

```diff
@@ -50,8 +50,30 @@ duplicates are present. We can add
 `rownames(mae[[1]]) <- getTaxonomyLabels(mae[[1]], make.unique=TRUE)` to
 remove them.
 
-```{r, message=FALSE, warning=FALSE}
+To run MOFA2, it needs to connect `mofapy2` Python packages via the `reticulate`
+package, as clarified in the [MOFA2 vignette](https://bioconductor.org/packages/release/bioc/vignettes/MOFA2/inst/doc/getting_started_R.html#6_Build_and_train_the_MOFA_object).
+
+The following chunk does not install and does not update `miniconda` if already
+installed. Furthermore, it does not install the latest version of `mofapy2`.
+If this setup causes issues, update `miniconda` and `mofapy2`.
+
+```{r}
+#| label: mofapy2
+#| message = false
+#| warning = false
+
+reticulate::install_miniconda(force = FALSE, update = FALSE)
+reticulate::use_miniconda(condaenv = reticulate::miniconda_path(), required = FALSE)
+reticulate::py_install(
+    packages = c("mofapy2"), pip = TRUE,
+    pip_ignore_installed = TRUE, python_version=3.12
+)
+```
+
+```{r}
```

Comment:
Can the installation be done just by specifying  `use_basilisk = TRUE`

https://rdrr.io/bioc/MOFA2/man/run_mofa.html